src/http/client.rs

```diff
@@ -1456,16 +1456,16 @@ impl Http {
         for file in files {
 
             match file.into() {
-                AttachmentType::Bytes((bytes, filename)) => {
+                AttachmentType::Bytes{data, filename} => {
```

Comment:
Formatting nitpick: Spaces after `{` and before `}`.
```suggestion
                AttachmentType::Bytes{ data, filename } => {
```